code/game/objects/items/weapons/blades.dm

```diff
@@ -32,6 +32,25 @@
 	icon_state = "machete"
 	w_class = SIZE_LARGE
 
+/obj/item/weapon/claymore/mercsword/machete/attack_self(mob/user)
+	if(user.action_busy)
+		return
+
+	var/turf/root = get_turf(user)
+	var/facing = user.dir
+	// List containing the 3 tiles in front of the user
+	var/list/in_front = list(get_step(root, facing),  get_step(root, turn(facing, 45)),  get_step(root, turn(facing, -45)))
```

Comment:
Definitely don't want this reaching out past the tile in front of the user, so just one tile there.